lib/Backend/GlobOpt.cpp

```diff
@@ -6957,6 +6995,10 @@ GlobOpt::SetValue(GlobOptBlockData *blockData, Value *val, Sym * sym)
     else
     {
         SetValueToHashTable(blockData->symToValueMap, val, sym);
+        if (blockData->changedSyms && isStackSym && sym->AsStackSym()->HasByteCodeRegSlot())
```

Comment:
Will do it.

---
In reply to: [68328865](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68328865) [](ancestors = 68328865)